src/net.c

```diff
@@ -1557,6 +1569,8 @@ handle_locally(coap_context_t *context __attribute__ ((unused)),
 handle_locally(coap_context_t *context, coap_queue_t *node) {
 #endif /* GCC */
   /* this function can be used to check if node->pdu is really for us */
+  (void)context;
```

Comment:
Indeed. It seemed a good idea at that time (2011!) but was never used for anything, so we can just get rid of it.